Pax.cs

```diff
@@ -185,7 +185,7 @@ private static void Configure(CaptureDeviceList devices)
             {
               PaxConfig.deviceMap[idx] = device;
               PaxConfig.rdeviceMap.Add(device.Name, idx);
-              device.Open();
+              device.Open(DeviceMode.Normal, 100); // 100ms timeout
```

Comment:
In the comment could you elaborate why the default timeout value isn't ideal please, and why 100 was chosen? (It's always a good idea to add a note indicating why a particular value. Sometimes this comment might simply say "fudge" because it's not based on much insight, but ideally it would say a little more.)